spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/security/oauth2/client/OAuth2SsoProperties.java

```diff
@@ -41,6 +44,88 @@
 	 */
 	private Integer filterOrder;
 
+	/**
+	 * If <code>true</code>, will always redirect to the value of {@link #succesDefaultTargetUrl}
+	 * (defaults to <code>false</code>).
+	 *
+	 * @see SavedRequestAwareAuthenticationSuccessHandler#setAlwaysUseDefaultTargetUrl(boolean)
+	 */
+	private boolean successAlwaysUseDefaultTargetUrl = false;
```

Comment:
does this apply to the description only (so no html and `@link` etc) or should I remove the `@see` too?